src/Model.js

```diff
@@ -142,12 +140,54 @@ export default class Model {
       : this.optimisticId
   }
 
+  /**
+   * Get an array with the attributes names that have changed.
+   */
+  @computed
+  get changedAttributes (): Array<string> {
+    return getChangedAttributesBetween(toJS(this.committedAttributes), toJS(this.attributes))
+  }
+
+  /**
+   * Gets the current changes.
+   */
+  @computed
+  get changes (): { [string]: mixed } {
+    return getChangesBetween(toJS(this.committedAttributes), toJS(this.attributes))
+  }
+
+  /**
+   * If an attribute is specified, returns true if it has changes.
+   * If no attribute is specified, returns true if any attribute has changes.
+   */
+  hasChanges (attribute?: string): boolean {
```

Comment:
Any reason this isn't @computed / get?

Reply:
Yeah, I agree. Maybe `get hasChanges()` and `hasAttributeChanged(attribute)`.